indent.sh

```diff
@@ -3,10 +3,22 @@
 # Indents all .c and .h files in the project that are not vendor code (ie: Unity).
 #
 
-INDENT=indent
+INDENT=""
+
+if [ -x "$(command -v indent)" ]; then
+  INDENT=indent
+fi
 
 if [ -x "$(command -v gindent)" ]; then
   INDENT=gindent
+elif [[ "$OSTYPE" == "darwin"* ]]; then
+  echo "ERROR: install gnu-indent using home brew to run this script."
+  exit 1
+fi
+
+if [[ -z "$INDENT" ]]; then
+  echo "Error:  Install indent to run this script."
+  exit 126
```

Comment:
Is there some significance to 126?

Reply:
Not sure if it was intended, but http://tldp.org/LDP/abs/html/exitcodes.html

Reply:
Yes,  I did read (google) the documentation.